infrastructure/storage-provisioning/README.md

```diff
@@ -85,3 +85,31 @@ After that you can delete test with commands
 $ kubectl delete -f examples/wordpress.yaml
 $ kubectl delete -f examples/mysql.yaml
 ```
+
+## Rook NFS
+The personal storage disk attached to VMs and containers is bassd on the NFS server provided by Rook-Ceph ([here](https://rook.io/docs/rook/v1.10/Storage-Configuration/NFS/nfs/)), which can be configured with the `CephNFS` CRD of the NFS-Ganesha server.
+
+The user's storage is dynamically provisioned through a `StorageClass` that contains a set of parameters that tell the Rook Operator to create also a NFS share; this is later mounted from user's VMs and containers though the NFS protocol.
+
```

Comment:
```suggestion
A single PV is created that contains all the user's personal space; then, multiple PVCs are created, one for each user, to mount the home folder in VMs and containers.
PVCs are created in a separate namespace that must already be present and the name of the namespace is passed as argument to the Tenant Operator.

```

Reply:
I don't think this is correct. One PV and one PVC are created for each user. These is not a single PV for all users.

Reply:
I understood that you created a single volume for all the users, isn't it?
Then, multiple PVC, one per user, to attach to the personal storage.
Am I wrong?

Reply:
We create a PVC for the user with a specified storage class, then the Rook operator provides a PV for that PVC dynamically. So at the end there is a PVC and a PV for each user.